tests/unit/test_routes.py

```diff
@@ -371,6 +371,66 @@ def test_update_resource(module_client, module_db, fake_auth_from_oc):
     assert (response.status_code == 404)
 
 
+def test_search(module_client, module_db, fake_auth_from_oc):
```

Comment:
`/search` isn't an authenticated endpoint, do we need the mock for fake_auth_from_oc?